pages/models.py

```diff
@@ -1272,7 +1272,7 @@ def get_context(self, *args, **kwargs):
 class GroupPage(RoutablePageMixin, Page):
     @route(r'^$')
     def index_view(self, request):
-        groups = Group.objects.all().order_by('state_organizing_committee')
+        groups = Group.objects.all().order_by('group_type')
```

Comment:
What is the purpose of the sorting here?

Reply:
So when results are returned in sidebar they are sorted by priority (state organizing committee first).

Reply:
But isn't it just going to sort alphabetically? I don't see where the group types are ranked in order of which should appear first. This looks like a string alpha sort?